root/static/scripts/common/i18n/semicolonOnlyList.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * @flow
+ * Copyright (C) 2015-2016 MetaBrainz Foundation
```

Comment:
I kept the copyright dates from `commaOnlyList` since it's basically 99% derived from that, not really anything new.

Re changing `commaOnlyList` to take an optional separator, it's possible, but it'd have to be a hardcoded list of separators (since it's fixed/set in the translated string), so we'd need a switch statement to pick the correct string and pass the correct interpolation argument names (semicolon_only_list_item vs. comma_only_list_item). TBH it also feels weird to allow that since "comma" is explicitly in the name (maybe we should just rename the function then, but).